webapp/src/js/services/update-service-worker.js

```diff
@@ -0,0 +1,50 @@
+/*
+Handles service worker updates
+*/
+angular.module('inboxServices').factory('UpdateServiceWorker', function($window, $log, $timeout) {
```

Comment:
I think you need to include the "ng-inject" directive otherwise these dependencies won't be injected in production.